src/main/app/claims/models/claim-states/part-admission-states.ts

```diff
@@ -0,0 +1,72 @@
+export enum PartAdmissionStates {
+  PART_ADMISSION = 'part-admission',
+  PA_PAY_IMMEDIATELY_WITH_MEDIATION = 'pa-pay-immediately-with-mediation',
+  PA_PAY_IMMEDIATELY_WITHOUT_MEDIATION = 'pa-pay-immediately-without-mediation',
+  PA_PAY_IMMEDIATELY_ACCEPTED_WITH_MEDIATION = 'pa-pay-immediately-accepted-with-mediation',
+  PA_PAY_IMMEDIATELY_ACCEPTED_WITHOUT_MEDIATION = 'pa-pay-immediately-accepted-without-mediation',
+  PA_PAY_IMMEDIATELY_REJECTED_WITH_MEDIATION_CLAIMANT_WITHOUT_MEDIATION = 'pa-pay-immediately-rejected-with-mediation-claimant-without-mediation',
+  PA_PAY_IMMEDIATELY_REJECTED_WITH_MEDIATION_CLAIMANT_WITH_MEDIATION = 'pa-pay-immediately-rejected-with-mediation-claimant-with-mediation',
+  PA_PAY_IMMEDIATELY_REJECTED_WITHOUT_MEDIATION = 'pa-pay-immediately-rejected-without-mediation',
+  PA_PAY_IMMEDIATELY_WITH_MEDIATION_PAST_DEADLINE = 'pa-pay-immediately-with-mediation-past-deadline',
+  PA_PAY_IMMEDIATELY_WITHOUT_MEDIATION_PAST_DEADLINE = 'pa-pay-immediately-without-mediation-past-deadline',
+  PA_CCJ_PAST_PAYMENT_DEADLINE_WITH_MEDIATION_BY_ADMISSION = 'pa-ccj-past-payment-deadline-with-mediation-by-admission',
+  PA_CCJ_PAST_PAYMENT_DEADLINE_WITHOUT_MEDIATION_BY_ADMISSION = 'pa-ccj-past-payment-deadline-without-mediation-by-admission',
+
+  PA_PAY_BY_SET_DATE_WITH_MEDIATION = 'pa-pay-by-set-date-with-mediation',
+  PA_PAY_BY_SET_DATE_WITHOUT_MEDIATION = 'pa-pay-by-set-date-without-mediation',
+  PA_PAY_BY_SET_DATE_REJECTED_WITH_MEDIATION_CLAIMANT_WITHOUT_MEDIATION = 'pa-pay-by-set-date-rejected-with-mediation-claimant-without-mediation',
+  PA_PAY_BY_SET_DATE_REJECTED_WITH_MEDIATION_CLAIMANT_WITH_MEDIATION = 'pa-pay-by-set-date-rejected-with-mediation-claimant-with-mediation',
+  PA_PAY_BY_SET_DATE_REJECTED_WITHOUT_MEDIATION = 'pa-pay-by-set-date-rejected-without-mediation',
+
+  PA_PAY_IN_INSTALMENTS_WITH_MEDIATION = 'pa-pay-in-instalments-with-mediation',
+  PA_PAY_IN_INSTALMENTS_WITHOUT_MEDIATION = 'pa-pay-in-instalments-without-mediation',
+  PA_PAY_IN_INSTALMENTS_REJECTED_WITH_MEDIATION_CLAIMANT_WITHOUT_MEDIATION = 'pa-pay-in-instalments-rejected-with-mediation-claimant-without-mediation',
+  PA_PAY_IN_INSTALMENTS_REJECTED_WITH_MEDIATION_CLAIMANT_WITH_MEDIATION = 'pa-pay-in-instalments-rejected-with-mediation-claimant-with-mediation',
+  PA_PAY_IN_INSTALMENTS_REJECTED_WITHOUT_MEDIATION = 'pa-pay-in-instalments-rejected-without-mediation',
+
+  PA_PAY_BY_SPECIFIED_DATE = 'pa-pay-by-specified-date',
```

Comment:
this doesn't seem to be used and I don't think it makes sense as you've covered everything above in the pay_by_set_datte